caravel/bin/caravel

```diff
@@ -36,8 +37,9 @@ manager.add_command('db', MigrateCommand)
     help="Specify the timeout (seconds) for the gunicorn web server")
 def runserver(debug, port, timeout, workers):
     """Starts a Caravel web server"""
-    debug = debug or config.get("DEBUG")
-    if debug:
+    debug = config.get('LOG_LEVEL')
```

Comment:
`debug` meant something else here (whether the web server should run in debug mode). We should keep that insulated from the logging level.